### PR TITLE
Add missing dependency on controller_manager_msgs

### PR DESCRIPTION
**Pull request by destogl**
Even after clearing the workspace and doing a clean build I had issues with missing dependency/fields when compiling this test.
It is obvious that the dependency is missing.

### CODE REVIEW

**Comment by mahaarbo** [comment]
This is related to https://github.com/ros-controls/ros2_control/issues/423 and solves the issue when tested on my end.

**Review by bmagyar — APPROVED** [review]
good spot, thanks!

File: controller_manager/CMakeLists.txt [hunk 1]
```diff
@@ -123,6 +123,10 @@ if(BUILD_TESTING)
   )
   target_include_directories(test_controller_manager_srvs PRIVATE include)
   target_link_libraries(test_controller_manager_srvs controller_manager test_controller)
+  ament_target_dependencies (
```
Comment (bmagyar) [inline controller_manager/CMakeLists.txt:126]:
```suggestion
  ament_target_dependencies(
```

**Comment by bmagyar** [comment]
ros2 signature keys expired, setup-ros scripts are failing on it. Will merge as industrial CI running these too passes

**Pull request merged**